packages/projects/src/components/ProjectForm.jsx

```diff
@@ -82,12 +83,12 @@ const reducer = (state, action) => {
 
 const formClass = 'mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-300 focus:ring focus:ring-indigo-200 focus:ring-opacity-50'
 
-const change = (dispatch, type, e) => {
+const change = (state, dispatch, type, e) => {
   dispatch({ type: 'formStatus', payload: FORM_STATUSES.clean })
 
   try {
     e.preventDefault()
-    const payload = e.target.value
+    const payload = e.target.value || null
```

Comment:
For `owner`, we initialize the default option's value as an empty string just because that's what the input wants. Assuming all the API calls go smoothly, the owner options would be populated so we might never hit the case where the `owner` value remains an empty string. But if we DID, it seems like it would be cleaner to explicitly have an empty owner set to `null`, since I don't know how the backend would handle an empty string `owner` and might consider it to be valid when it shouldn't.